### PR TITLE
Change to billboard js chart

### PR DESCRIPTION
**Pull request by imbyungjun**

### CODE REVIEW

File: ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestService.java [hunk 1]
```diff
@@ -1185,29 +1184,29 @@ public int getMonitorGraphInterval(long testId, String targetIP, int imageWidth)
 					skipCount = 0;
 					String[] datalist = StringUtils.split(line, ",");
 					if ("null".equals(datalist[4]) || "undefined".equals(datalist[4])) {
```
Comment (donggyu04) [inline ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestService.java]: use `NULL_STRING` or "null" 
I think it's good to unite in one way.

Comment (imbyungjun) [reply ngrinder-controller/src/main/java/org/ngrinder/perftest/service/PerfTestService.java]: I didn't find out there's a code uses the string value "null". I'm gonna change it to use string constants

File: ngrinder-frontend/src/js/components/perftest/detail/RampUp.vue [hunk 2]
```diff
@@ -159,118 +181,150 @@
...
 <style lang="less" scoped>
+    @import '~billboard.js/dist/billboard.min.css';
+    @import '~billboard.js/dist/theme/insight.min.css';
+
     .ramp-up-container {
         width: 460px;
         margin-left: 18px;
 
+        .enable-checkbox {
+            vertical-align: text-bottom;
```
Comment (donggyu04) [inline ngrinder-frontend/src/js/components/perftest/detail/RampUp.vue]: Why don't you use bootstrap predefined class?

like this `<td class="align-text-bottom">text-bottom</td>`

**Comment by junoyoon** [comment]
LGTM

**Pull request merged**